### PR TITLE
[Klaudsol-CMS-Extra] Fix s3 access keys

### PR DESCRIPTION
**Pull request by JohnServo24**
Change environment variables.

I created a branch from `main` and pulled from `staging`. There are 38 files changed, but I only changed 2 files. `S3.js` and `EntityAttributeValue.js`. But I believe that the changes on the other files are up to date.

### CODE REVIEW

File: backend/data_access/S3.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import AWS from "aws-sdk";
+import { promisify } from "es6-promisify";
+import crypto from "crypto";
+
+const S3_ACCESS_KEY_ID = process.env.S3_ACCESS_KEY_ID;
+const S3_SECRET_ACCESS_KEY = process.env.S3_SECRET_ACCESS_KEY;
```
Comment (ardeearam) [inline backend/data_access/S3.js:6]:
```suggestion
//TODO: Deprecate AURORA_AWS_ACCESS_KEY_ID & AURORA_AWS_SECRET_ACCESS_KEY
const S3_ACCESS_KEY_ID = process.env.KS_S3_ACCESS_KEY_ID ??  process.env.KS_AWS_ACCESS_KEY_ID ?? process.env.AURORA_AWS_ACCESS_KEY_ID;
const S3_SECRET_ACCESS_KEY = process.env.KS_S3_SECRET_ACCESS_KEY ?? process.env.KS_AWS_SECRET_ACCESS_KEY ?? process.env.AURORA_AWS_SECRET_ACCESS_KEY;
```

File: backend/data_access/S3.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+const REGION = process.env.AURORA_AWS_REGION;
```
Comment (ardeearam) [inline backend/data_access/S3.js:7]:
```suggestion
//TODO: Deprecate AURORA_AWS_REGION on release V3.0.0
const REGION = process.env.KS_S3_REGION ?? process.env.KS_AWS_REGION ?? process.env.AURORA_AWS_REGION;
```

File: backend/data_access/S3.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+const S3_BUCKET = process.env.S3_BUCKET;
```
Comment (ardeearam) [inline backend/data_access/S3.js:8]:
```suggestion
const S3_BUCKET = process.env.KS_S3_BUCKET;
```

File: components/EntityAttributeValue.js [hunk 4]
```diff
@@ -1,13 +1,23 @@
...
       case 'text':
-      case 'image':
       case 'link':
         return item.value_string;
       case 'textarea':
         return item.value_long_string;
+      case 'image':
+        if(!item.value_string) return;
+        
+        const bucketBaseUrl = process.env.S3_BASE_URL;
```
Comment (ardeearam) [inline components/EntityAttributeValue.js:11]:
```suggestion
        const bucketBaseUrl = process.env.KS_S3_BASE_URL;
```

File: lib/Discord.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 export async function discordSay(message) {
 
-    const token = process.env.DISCORD_TOKEN;
+    //TODO: Deprecate DISCORD_TOKEN
```
Comment (ardeearam) [inline lib/Discord.js:3]:
```suggestion
    //TODO: Deprecate DISCORD_TOKEN in release v3.0.0
```

File: next.config.js [hunk 6]
```diff
@@ -2,8 +2,17 @@
...
   productionBrowserSourceMaps: true,
+  env: {
+    AURORA_AWS_ACCESS_KEY_ID: process.env.AURORA_AWS_ACCESS_KEY_ID,
+    AURORA_AWS_SECRET_ACCESS_KEY: process.env.AURORA_AWS_SECRET_ACCESS_KEY,
+    AURORA_DATABASE: process.env.AURORA_DATABASE,
+    AURORA_RESOURCE_ARN: process.env.AURORA_RESOURCE_ARN,
+    AURORA_SECRET_ARN: process.env.AURORA_SECRET_ARN,
+    SECRET_COOKIE_PASSWORD: process.env.SECRET_COOKIE_PASSWORD,
+    FRONTEND_URL: process.env.FRONTEND_URL,
+  },
```
Comment (ardeearam) [inline next.config.js:13]: This is already removed. All succeeding environment variable changes should be placed in `amplify.yml`

```suggestion
  },
```

File: next.config.js [hunk 7]
```diff
@@ -2,8 +2,17 @@
...
   images: {
-    domains: (process.env.IMAGE_DOMAINS ?? "").split(",")
+    domains: (process.env.IMAGE_DOMAINS ?? "").split(","),
```
Comment (ardeearam) [inline next.config.js:15]:
```suggestion
    domains: (process.env.KS_IMAGE_DOMAINS ?? "").split(","),
```

**Review by ardeearam — CHANGES_REQUESTED** [review]
Kindly see the change requests to adhere to our new environment variable naming convention.

**Review by angeli-mercedes — APPROVED** [review]

**Pull request merged**